lib/lists.nix

```diff
@@ -409,6 +411,25 @@ rec {
               then compareLists cmp (tail a) (tail b)
               else rel;
 
+  /* Sort list using "Natural sorting".
+     Numeric portions of strings are sorted in numeric order.
+
+     Example:
+       naturalSort ["disk11" "disk8" "disk100" "disk9"]
+       => ["disk8" "disk9" "disk11" "disk100"]
+       naturalSort ["46.133.149.113" "5.16.62.13" "54.16.25.114"]
+       => ["5.16.62.13" "46.133.149.113" "54.16.25.114"]
```

Comment:
Minor nitpick: I think it would be better to not use valid public ip addresses here. Here are 3 private ones instead: `["10.222.250.51" "172.20.255.255" "192.168.124.49" ]`

Reply:
They are not valid, just random;
but I agree, it might cause some problems to their maintainers

Reply:
According to whois, those IPs belong to Z-Telecom Network in Russia, Ukrainian Mobile Communications and Merck and Co., Inc. in NJ US  :)

Reply:
Oh, there was a chance to feed journos by using IP of The Ecuadorian Embassy or something :)